src/lib.rs

```diff
@@ -481,67 +755,163 @@ impl<I: Interchange> Responder<I> {
         {
             Ok(())
         } else {
-            Err(())
+            Err(Error)
         }
     }
 
     /// Respond to a request.
     ///
     /// If efficiency is a concern, or responses need multiple steps to
-    /// construct, use `response_mut` and `send_response.
+    /// construct, use `with_response_mut` or `response_mut` and `send_response`.
```

Comment:
s/response_mut/response/

Reply:
I've renamed the method to `response_mut` for consistency with the requester